frontend/src/modules/history/components/History.js

```diff
@@ -21,7 +21,7 @@ type Props = {|
     locale: Locale,
     user: UserState,
     deleteTranslation: (number) => void,
-    addComment: (string, number) => void,
+    addComment: (string, any) => void,
```

Comment:
Maybe @adngdb has an idea why `?number` makes flow unhappy here and also in `Helpers.js`?
```
Cannot create Translation element because number [1] is incompatible
with null or undefined [2] in the second argument of property addComment.
```

Reply:
Because the function is defined like this:

```js
addComment = (comment: string, translation?: number) => {
```

and instead it should be defined like this:

```js
addComment = (comment: string, translation: ?number) => {
```

However, please don't ask me why, as I have no freaking clue. :/